content/en/docs/topics/advanced.md

```diff
@@ -5,69 +5,66 @@ aliases: ["/docs/advanced_helm_techniques"]
+This section explains various advanced features and techniques for using Helm. The information in
+this section is intended for "power users" of Helm that wish to do advanced customization and
+manipulation of their charts and releases. Each of these advanced features comes with their own
+tradeoffs and caveats, so each one must be used carefully and with deep knowledge of Helm. Or in
+other words, remember the [Peter Parker
 principle](https://en.wikipedia.org/wiki/With_great_power_comes_great_responsibility)
 
 ## Post Rendering
-Post rendering gives chart installers the ability to manually manipulate,
-configure, and/or validate rendered manifests before they are installed by Helm.
-This allows users with advanced configuration needs to be able to use tools like
-[`kustomize`](https://kustomize.io) to apply configuration changes without the
-need to fork a public chart or requiring chart maintainers to specify every last
-configuration option for a piece of software. There are also use cases for
-injecting common tools and side cars in enterprise environments or analysis of
-the manifests before deployment.
+
+Post rendering gives chart installers the ability to manually manipulate, configure, and/or validate
+rendered manifests before they are installed by Helm. This allows users with advanced configuration
+needs to be able to use tools like [`kustomize`](https://kustomize.io) to apply configuration
+changes without the need to fork a public chart or requiring chart maintainers to specify every last
+configuration option for a piece of software. There are also use cases for injecting common tools
+and side cars in enterprise environments or analysis of the manifests before deployment.
 
 ### Prerequisites
+
 - Helm 3.1+
 
 ### Usage
-A post-renderer can be any executable that accepts rendered Kubernetes manifests
-on STDIN and returns valid Kubernetes manifests on STDOUT. It should return an
-non-0 exit code in the event of a failure. This is the only "API" between the two
-components. It allows for great flexibility in what you can do with your
-post-render process.
 
-A post renderer can be used with `install`, `upgrade`, and `template`. To use a
-post-renderer, use the `--post-renderer` flag with a path to the renderer
-executable you wish to use:
+A post-renderer can be any executable that accepts rendered Kubernetes manifests on STDIN and
+returns valid Kubernetes manifests on STDOUT. It should return an non-0 exit code in the event of a
+failure. This is the only "API" between the two components. It allows for great flexibility in what
+you can do with your post-render process.
+
+A post renderer can be used with `install`, `upgrade`, and `template`. To use a post-renderer, use
+the `--post-renderer` flag with a path to the renderer executable you wish to use:
 
 ```shell
-$ helm install mychart stable/wordpress --post-renderer ./path/to/executable
+helm install mychart stable/wordpress --post-renderer ./path/to/executable
```

Comment:
This led me to look into the difference between shell and console in markdown. While they display similarly on the website and on github (not showing the highlighting variation that is possible), the linter approves of `$` with console and not with shell. Do we want to standardize on either shell or console?

Reply:
I did a little digging. The `shell` type is for shells... config details at https://github.com/github/linguist/blob/b24ff11d9cf60dd996156a718f83024c9436e5ea/lib/linguist/languages.yml#L5043

The `console` type is for Shell Sessions. It's an alias. Appears to use the same syntax highlighting as the `shell` type. Config at https://github.com/github/linguist/blob/b24ff11d9cf60dd996156a718f83024c9436e5ea/lib/linguist/languages.yml#L5108.

Due to this, I would standardize on shell.